src/guides/v2.3/install-gde/continue-to-verify_cli.md

```diff
@@ -2,7 +2,7 @@
 group: installation-guide
 subgroup: W_continue
 title:
-menu_title: Next&mdash;Installation part 3, post-installation
+menu_title: Next---Installation part 3, post-installation
```

Comment:
Revert this. Long term, it would be better if the title were changed so that that there's no em dash.

Reply:
**done**